app/views/admin/settings/forms/authentication/_apple_auth_provider_settings.html.erb

```diff
@@ -33,10 +33,10 @@ requires a custom partial
     <p class="crayons-field__description">
       <%= Constants::Settings::Authentication::DETAILS[:apple_pem][:description] %>
     <p>
-    <%= f.text_field :apple_pem,
-                     class: "crayons-textfield",
-                     value: Settings::Authentication.apple_pem,
-                     placeholder: Constants::Settings::Authentication::DETAILS[:apple_pem][:placeholder] %>
+    <%= f.text_area :apple_pem,
+                    class: "crayons-textfield",
+                    value: Settings::Authentication.apple_pem,
+                    placeholder: Constants::Settings::Authentication::DETAILS[:apple_pem][:placeholder] %>
```

Comment:
**non-blocking suggestion:** By making the default height a bit taller we make the entire PEM file (it's generally quite short) visible in it's entirety. The positive impact here is that the creator would be able to take a quick glance at it and ensure it's formatted correctly (check newlines are indeed there) regardless of which style they used.

```suggestion
                    placeholder: Constants::Settings::Authentication::DETAILS[:apple_pem][:placeholder],
                    style: "height: 175px;"%>
```

What this looks like:

<img width="1070" alt="Screen Shot 2021-09-16 at 10 34 50" src="https://user-images.githubusercontent.com/6045239/133651250-beb7844c-4a51-4f54-b2b6-903d9bf382e4.png">

Reply:
Would it make sense to set an attribute `rows=6` (how large should the text area be, in lines of text), rather than override the style using a pixel size? I'm not sure what that would mean for users with either very large text or limited screen area, it's probably equivalent in many cases.